### ISSUE TITLE
can we add some commonly used software acronyms in this modern dictionary?

### ISSUE THREAD

**Issue by sridhar02** [tatianamac/selfdefined#254]
* FOMO - Fear of Missing Out
* EOD - End of the day
* MVP - minimum viable project
etc ...

**Comment by tatianamac** [comment]
Hi @sridhar02 ! Acronyms are definitely hard to parse for folks who are unfamiliar. With that said, I'm not sure these acronyms fit into the overarching scope of this dictionary. I'll share with the group to see if they have good suggestions as to where to put these instead or if there's another, more appropriate resource for them!
